docroot/themes/contrib/civictheme/includes/migrate.inc

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/**
+ * @file
+ * Helper functions adding CivicTheme migration metadata.
+ *
+ * Information non-visible content data including theme,
+ * vertical_spacing, filter options etc.
+ */
+
+use Drupal\media\MediaInterface;
```

Comment:
Next step for this exposing metadata is to expose a data-attribute for each component in civictheme_page
like `data-ct-migrate-component="<component_type"` that way we have a clear API for merlin to target to extract the paragraph components as opposed to when we reuse twig templates with the same html structure ie basic content twig gets reused by a lot of contents